instances/latest/brainAtlasVersions/MarmosetNMA/MarmosetNMA_v1.jsonld

```diff
@@ -0,0 +1,407 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.om-i.org/props/"
+    },
+    "@id": "https://openminds.om-i.org/instances/brainAtlasVersion/MarmosetNMA_v1",
+    "@type": "https://openminds.om-i.org/types/BrainAtlasVersion",
+    "abbreviation": "MarmosetNMA",
+    "author": null,
+    "custodian": null,
+    "description": "The Marmoset Nencki-Monash Atlas is a three dimensional (3D) probabilistic brain atlas reconstructed from 20 young adult marmoset monkeys (Callithrix jacchus) and segmented based on the cytoarchitectonic analysis of the serial Nissl-stained sections of those brains.",
+    "digitalIdentifier": {
```

Comment:
note: this is a place holder. Needs to be replaced with DOI for dataset version for the atlas/parcellation (once this is out)